misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/MaterialReceiptLineScreen.jsx

```diff
@@ -4,10 +4,33 @@ import { push, go } from 'connected-react-router';
 import counterpart from 'counterpart';
 
 import { updateManufacturingReceiptQty, updateManufacturingReceipt } from '../../../actions/ManufacturingActions';
+import { pushHeaderEntry } from '../../../actions/HeaderActions';
 import PickQuantityButton from './PickQuantityButton';
 import { toastError } from '../../../utils/toast';
+import { getLocation } from '../../../utils';
 
 class MaterialReceiptLineScreen extends PureComponent {
+  componentDidMount() {
+    const {
+      dispatch,
+      lineProps: { productName },
+    } = this.props;
+    const location = getLocation(this.props);
```

Comment:
pls create a function which creates the location for a manufacturing receipt line and which takes specific params.